packages/wallets/src/features/accounts/screens/IDVDocumentUpload/IDVDocumentUpload.tsx

```diff
@@ -43,15 +60,25 @@ const IDVDocumentUpload = () => {
                 <div className='wallets-idv-document-upload__title'>
                     <WalletText weight='bold'>Identity verification</WalletText>
                 </div>
-                {/* TODO: Update dropdown after Formik is implemented */}
                 <WalletDropdown
                     label='Choose the document type'
-                    list={[]}
-                    name='wallets-idv-document-upload__dropdown'
-                    onSelect={() => null}
+                    list={[
+                        { text: 'Drivers License', value: 'driverLicense' },
+                        { text: 'Passport', value: 'passport' },
+                        { text: 'Social Security and National Insurance Trust (SSNIT)', value: 'ssnit' },
+                    ]}
+                    name='documentType'
+                    onSelect={selectedItem => {
+                        setSelectedDocument(selectedItem);
+                        setFormValues('documentType', selectedItem);
+                    }}
                     value={undefined}
                 />
-                <FlowTextField label='Enter your document number' name='documentNumber' />
+                <FlowTextField
+                    label='Enter your document number'
+                    name='documentNumber'
+                    validationSchema={validationSchema}
```

Comment:
Explanation: We added an option to pass in our own validators to text field component, since there are cases where we have different validators for different documents like this

I.e. When user selects passport, we need to use `passportValidator`, when user selects driving license, we need to use `drivingLicenseValidator`